connection.go

```diff
@@ -735,6 +735,10 @@ func buildSnowflakeConn(ctx context.Context, config Config) (*snowflakeConn, err
 		queryContextCache:   (&queryContextCache{}).init(),
 		currentTimeProvider: defaultTimeProvider,
 	}
+	err := initEasyLogging(config.ClientConfigFile)
+	if err != nil {
+		return nil, err
```

Comment:
Should we fail here? Maybe warn is enough? This is far from being a critical feature.

Reply:
we do fail in other drivers as well. As I remember it was supposed to work this way.

Reply:
I changed the exception to be more about "client config" than about "easy logging"